feat: add title and description to the frontmatter of mdx docs (#154)
clerkinc/clerk-docs: docs/advanced-usage/custom-session-token.mdx

### Add a new claim to the session token

> At this point you are able to select all the session tokens you need. In this example, we are adding a new claim called `fullName` and `primaryEmail` to the session token.

Author: ```suggestion
At this point you are able to select all the session tokens you need. This example adds a new claim called `fullName` and `primaryEmail` to the session token.
```
